### PR TITLE
Add `IndexFromAccessor`

### PR DESCRIPTION
**Pull request by lilleyse**
Adds a visitor that retrieves the index supplied by `IndexAccessorType`. There's some overlap with `IndicesForFaceFromAccessor ` but this allows for simpler direct access of indices.

### CODE REVIEW

File: CesiumGltf/include/CesiumGltf/AccessorUtility.h [hunk 1]
```diff
@@ -229,6 +229,28 @@ struct IndicesForFaceFromAccessor {
   int32_t primitiveMode;
 }; // namespace CesiumGltf
 
+/**
+ * Visitor that retrieves the vertex index from the given accessor type as an
+ * int64_t. This should be initialized with the index of the vertex index.
```
Comment (j9liu) [inline CesiumGltf/include/CesiumGltf/AccessorUtility.h:234]: Hm is there a clearer way to word "index of the vertex index"? 
```suggestion
 * int64_t. This should be initialized with the index (within the 
 * accessor itself) of the vertex index.
```

**Review by j9liu — COMMENTED** [review]
@lilleyse looks good to me! Just had one comment to improve clarity

**Comment by lilleyse** [comment]
Thanks for the suggestion! This should be ready now.

**Comment by kring** [comment]
Merging this so it can go in today's cesium-native release. Thanks @lilleyse!

**Pull request merged**